src/github.com/getlantern/peerscanner/web.go

```diff
@@ -123,14 +138,28 @@ func getHostInfo(req *http.Request) (name string, ip string, port int, err error
 		return
 	}
 	portString := req.FormValue("port")
-
 	if portString != "" {
 		port, err = strconv.Atoi(portString)
 		if err != nil {
 			err = fmt.Errorf("Received invalid port for %v - %v: %v", name, ip, portString)
 		}
 	}
-
+	fronts := req.FormValue("fronts")
+	if fronts == "" {
+		// backwards compatibility
+		fronts = "cloudflare"
+	}
+	for _, front := range strings.Split(fronts, ",") {
```

Comment:
This is a bug; I need to `req.ParseForm()` and use `fronts := req.Form["fronts"]` as a `[]string`.  I have fixed this in a local branch (I'm not pushing it yet because it would bring the AWS SDK).